SimTracker/SiPhase2Digitizer/plugins/PixelDigitizerAlgorithm.cc

```diff
@@ -134,9 +134,12 @@ void PixelDigitizerAlgorithm::add_cross_talk(const Phase2TrackerGeomDetUnit* pix
   // cross-talk calculation valid for the case of 25x100 pixels
   const float pitch_first = 0.0025;
   const float pitch_second = 0.0100;
+  const double pitch_tolerance(
+      0.0005);  // 0.5 um tolerance when comparing the pitch to accommodate the small changes in different TK geometrie (temporary fix)
```

Comment:
it's better to put the comment on a separate line so the code format doesn't split the line in an awkward way like this